java/viewshed-geoprocessing/src/main/res/layout/activity_main.xml

```diff
@@ -12,4 +12,30 @@
         android:layout_height="match_parent">
     </com.esri.arcgisruntime.mapping.view.MapView>
```

Comment:
Not anything you did, but to clean up here we can just end the MapView block with />